src/main/resources/edu/wpi/first/pathweaver/welcomeScreen.fxml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.ListView?>
+<?import javafx.scene.control.TitledPane?>
+<?import javafx.scene.layout.BorderPane?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.text.Font?>
+
+<BorderPane fx:id="borderPane" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="768.0" prefWidth="1024.0" xmlns="http://javafx.com/javafx/10.0.1" xmlns:fx="http://javafx.com/fxml/1" fx:controller="edu.wpi.first.pathweaver.WelcomeController">
```

Comment:
Odd size, lots of empty space especially on the "Create Project" screen

Reply:
So, I agree Create Project needs to be expanded to fill the screen more. The issue we were running into with the previous size was that it was sort of small. We don't want to make it too large in case teams are using old laptops. I'm open to suggestions on what you'd recommend for the default size.